core/chains/evm/client/errors.go

```diff
@@ -123,20 +126,21 @@ var arbitrumFatal = regexp.MustCompile(`(: |^)(invalid message format|forbidden
 var arbitrum = ClientErrors{
 	// TODO: Arbitrum returns this in case of low or high nonce. Update this when Arbitrum fix it
 	// https://app.shortcut.com/chainlinklabs/story/16801/add-full-support-for-incorrect-nonce-on-arbitrum
-	NonceTooLow: regexp.MustCompile(`(: |^)invalid transaction nonce$|(: |^)nonce too low(:|$)`),
-	// TODO: Is it terminally or replacement?
+	NonceTooLow:           regexp.MustCompile(`(: |^)invalid transaction nonce$|(: |^)nonce too low(:|$)`),
 	TerminallyUnderpriced: regexp.MustCompile(`(: |^)gas price too low$`),
 	InsufficientEth:       regexp.MustCompile(`(: |^)(not enough funds for gas|insufficient funds for gas \* price \+ value)`),
 	Fatal:                 arbitrumFatal,
+	L2FeeTooLow:           regexp.MustCompile(`(: |^)max fee per gas less than block base fee(:|$)`),
```

Comment:
The `max fee per gas less than block base fee` can also be extended to other clients as well (not L2 specifically). i.e. geth uses the same error for EIP-1559 transactions: https://github.com/ethereum/go-ethereum/blob/c394c308e6211e3d09d608f0f1042f1637c83706/core/error.go#L95